tests/e2e/support/objects/app-files/page/public.ts

```diff
@@ -72,6 +72,15 @@ export class Public {
     await this.#page.locator('body').click()
   }
 
+  async uploadInternal(args: Omit<uploadResourceArgs, 'page'> & { link: string }): Promise<void> {
+    // link is the public link url
+    const { link } = args
+    delete args.link
```

Comment:
why delete this?

Reply:
link is not needed for the next function. So, extracted it and then deleted from the args list